docs/pages/kubernetes-access/troubleshooting.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: Kubernetes Access Troubleshooting Guide
+description: Troubleshooting common issues with Kubernetes access
+---
+
+## An agent failed to join a cluster due to "no authorities for hostname"
+
+### Symptoms
+
+The agent can't rejoin the Teleport cluster after restart and reports an error
+similar to:
+
+```text
+ssh: handshake failed: ssh: no authorities for hostname
+```
+
+### Explanation
+
+Teleport uses Certificate Authorities (CAs) to sign certificates for each
+component. When the component joins the cluster for the first time, it
+receives a certificate signed by the cluster's CA and stores it in its
+state directory. When the component restarts, it uses the certificate stored in
+its state directory to join the cluster again.
+
+This error occurs when the component tries to rejoin the Teleport cluster, but the
+cluster's CA has changed and the component's certificate is no longer valid.
+
+It can happen when the cluster's CA is rotated or when the cluster is recreated
+or renamed.
+
+### Resolution
+
+The agent's state needs to be reset, so it requests a new certificate and joins
+the cluster again.
```

Comment:
As written this could be read as "because it needs to be reset, it requests blah blah on it's own. Suggested rephrase:

```suggestion
The agent's state needs to be reset, so it can request a new certificate and join
the cluster again.
```